lib/Page.js

```diff
@@ -694,6 +694,41 @@ class Page extends EventEmitter {
   }
 
   /**
+   * @param {string} selector
+   * @param {!Array<string>} values
+   */
+  async select(selector, ...values) {
+    const handle = await this.$(selector);
+    console.assert(handle, 'No node found for selector: ' + selector);
+
+    await this.evaluate((element, values) => {
+      if (element.nodeName.toLowerCase() !== 'select')
```

Comment:
Why call `toLowerCase` here? Checking against `'SELECT'` is just fine and saves extra work from happening.